boards/riscv32/rv32m1_vega/doc/rv32m1_vega.rst

```diff
@@ -0,0 +1,782 @@
+
+===   ========   =================
+Pin   Port/bit   Function
+===   ========   =================
+1     D/5        FLEXIO_D25
+3     D/4        FLEXIO_D24
+5     D/3        FLEXIO_D23
+7     D/2        FLEXIO_D22
+9     D/1        FLEXIO_D21
+11    D/0        FLEXIO_D20
+13    C/30       FLEXIO_D19
+15    C/29       FLEXIO_D18
+17    C/28       FLEXIO_D17
+19    B/29       FLEXIO_D16
+===   ========   =================
+
+Even/top pins:
+
+===   ========   =================
+Pin   Port/bit   Function
+===   ========   =================
+2     B/2        GPIO
+4     B/3        PWM
+6     B/6        SPI0_PCS2
+8     B/5        SPI0_SOUT
+10    B/7        SPI0_SIN
+12    B/4        SPI0_SCK
+14    -          GND
+16    -          AREF
+18    C/9        I2C2_SDA
+20    C/10       I2C2_SCL
+===   ========   =================
+
+**Bottom left expansion header (J3)**
+
+Note that the headers at the bottom of the board have odd-numbered
+pins on the top, unlike the headers at the top of the board.
+
+Odd/top pins:
+
+===   ========   ====================
+Pin   Port/bit   Function
+===   ========   ====================
+1     A/21       ARDUINO_EMVSIM_PD
+3     A/20       ARDUINO_EMVSIM_IO
+5     A/19       ARDUINO_EMVSIM_VCCEN
+7     A/18       ARDUINO_EMVSIM_RST
+9     A/17       ARDUINO_EMVSIM_CLK
+11    B/17       FLEXIO_D7
+13    B/16       FLEXIO_D6
+15    B/15       FLEXIO_D5
+===   ========   ====================
+
+Even/bottom pins: note that these are mostly power-related.
+
+===   ========   =================
+Pin   Port/bit   Function
+===   ========   =================
+2     -          SDA_GPIO0
+4     -          BRD_IO_PER
+6     -          RST_SDA
+8     -          BRD_IO_PER
+10    -          P5V_INPUT
+12    -          GND
+14    -          GND
+16    -          P5-9V VIN
+===   ========   =================
+
+**Bottom right expansion header (J4)**
+
+Note that the headers at the bottom of the board have odd-numbered
+pins on the top, unlike the headers at the top of the board.
+
+Odd/top pins:
+
+===   ========   ========================================
+Pin   Port/bit   Function
+===   ========   ========================================
+1     -          TAMPER2
+3     -          TAMPER1/RTC_CLKOUT
+5     -          TAMPER0/RTC_WAKEUP_b
+7     E/2        ADC0_SE19
+9     E/5        LPCMP1_IN2/LPCMP1_OUT
+11    -          DAC0_OUT/ADC0_SE16/LPCMP0_IN3/LPCMP1_IN3
+===   ========   ========================================
+
+Even/bottom pins:
+
+===   ========   ===========================================
+Pin   Port/bit   Function
+===   ========   ===========================================
+2     C/11       ADC0_SE6
+4     C/12       ADC0_SE7
+6     B/9        ADC0_SE3
+8     E/4        ADC0_SE21
+10    E/10       ADC0_SE19 (and E/10, I2C_SDA via 0 Ohm DNP)
+12    E/11       ADC0_SE20 (and E/11, I2C_SCL via 0 Ohm DNP)
+===   ========   ===========================================
+
+Additional Pins
+~~~~~~~~~~~~~~~
```

Comment:
Please use our document conventions and use ``------`` here and below instead of ``~~~~~~`` for this heading level.